sdks/java/io/rrio/src/test/java/org/apache/beam/io/requestresponse/EchoGRPCCallerWithSetupTeardownIT.java

```diff
@@ -50,11 +50,11 @@ public class EchoGRPCCallerWithSetupTeardownTestIT {
   @BeforeClass
   public static void setUp() throws UserCodeExecutionException {
     options = readIOTestPipelineOptions(EchoITOptions.class);
-    if (options.getgRPCEndpointAddress().isEmpty()) {
+    if (options.getGrpcEndpointAddress().isEmpty()) {
       throw new RuntimeException(
           "--gRPCEndpointAddress is missing. See " + EchoITOptions.class + "for details.");
```

Comment:
nit: `--grpcEndpointAddress`

Reply:
Good call. I changed the option but not the comment on it.